src/index.ts

```diff
@@ -172,6 +172,11 @@ interface EventLogger {
    */
   logImpression(impression: Impression): void;
 
+  /**
+   * Logs `action`.
+   */
+  logAction(action: Action): void;
```

Comment:
Just for my education, this just exposes a method that's implemented somewhere else, right?

Reply:
Yea.  The impl class already supports it.  I don't know why I didn't expose it before.